src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2343,6 +2358,70 @@ MavlinkReceiver::handle_message_hil_state_quaternion(mavlink_message_t *msg)
 	}
 }
 
+void MavlinkReceiver::handle_message_named_value_float(mavlink_message_t *msg)
```

Comment:
The point is to transfer transparently debug messages between MAVLink and uORB. 
So yes, with this addition you can do it. You could implement it as a simple protocol, with your module waiting for a "SEND_XXX" message, answering with a continuous stream of "XXX" messages along with the desired value, until it receives a "STOP_XXX" message. That would be quite inefficient as it involves string comparisons, but ok in the development/debug phase.